cmd/boulder/main.go

```diff
@@ -75,11 +75,14 @@ func main() {
 
 		// Create the components
 		wfei := wfe.NewWebFrontEndImpl()
+		wfei.AccountKeyRateLimit.Resize(cmd.RateLimitWidth, c.WFE.AccountKeyRequestsPerSecond, time.Second)
+
 		sa, err := sa.NewSQLStorageAuthority(c.SA.DBDriver, c.SA.DBName)
 		cmd.FailOnError(err, "Unable to create SA")
 		sa.SetSQLDebug(c.SQL.SQLDebug)
 
 		ra := ra.NewRegistrationAuthorityImpl()
+		ra.DomainRateLimit.Resize(cmd.RateLimitWidth, c.RA.DomainRequestsPerSecond, time.Second)
```

Comment:
Second is probably the wrong unit here. It means the most aggressive rate limit we can apply is 1 subdomain request per second, when we probably want something more like 20 per day (absent an override to allow for more).